shared/components/CreateForm.js

```diff
@@ -14,7 +14,7 @@ import TForm from 'tcomb-form-native';
 import RNFS from 'react-native-fs';
 import {Actions} from 'react-native-router-flux';
 
-const API_SETITEMS = 'http://goober.herokuapp.com/api/items';
+const API_SETITEMS = 'http://ec2-52-78-96-168.ap-northeast-2.compute.amazonaws.com/api/items';
```

Comment:
components/CreateForm.js is outdated and removed. please see components/Create.js